cubeb-core/src/stream.rs

```diff
@@ -93,14 +93,9 @@ impl StreamParamsRef {
     }
 }
 
-unsafe fn wrapped_cubeb_stream_destroy(stream: *mut ffi::cubeb_stream) {
-    ffi::cubeb_stream_stop(stream);
-    ffi::cubeb_stream_destroy(stream);
-}
-
 ffi_type_heap! {
     type CType = ffi::cubeb_stream;
-    fn drop = wrapped_cubeb_stream_destroy;
+    fn drop = ffi::cubeb_stream_destroy;
```

Comment:
This means it's (again) unsafe for a core::Stream to drop if it's running, which seems like a bad thing.